docs/api/schema.yml

```diff
@@ -2760,9 +2773,9 @@ components:
           type: integer
```

Comment:
Hmm, `assigned_dev` is being picked up as an integer field when it's actually a nested field on read but an integer field on write. I'll see if I can find a way to annotated it correctly with `drf-spectacular`